tests/test_raster.py

```diff
@@ -187,13 +188,21 @@ def test_get_dask_array(self):
         self.assertIsInstance(img.to_dask_array(chunk_size=(1, 10, 10)), dask.array.core.Array)
 
     def test_write_to_file(self):
-        img.write_to_file(r"result.tif")
+        img.write_to_file(r"result.tif", rasterio.uint16)

```

Comment:
here you should also be able to use numpy dtype like np.uint16. then no need to import rasterio.

Reply:
Okay: 07ff06c